### ISSUE TITLE
Broken by 10.10.4?

### ISSUE THREAD

**Issue by sparky672** [dokterdok/Continuity-Activation-Tool#192]
I had everything working for a few months on my 2012 Mac Pro using a dongle.  No problems.

Now today Handoff is totally broken in both directions.  The only recent change was a system update to 10.10.4.  I am logged into iCloud and there have been no recent changes with that.

The CAT tool is telling me "OK" on everything under option 2 (Diagnostic) and says CAT is already activated when I try option 1 (Activate).

Anybody else?

**Comment by cohilla** [comment]
Did you reinstall the Tool? It need to be reinstalled after every OS update.

**Comment by sparky672** [comment]
I did not reinstall the tool because CAT says it's already installed.  Besides, I did not have to reinstall anything after the last two OS updates.

Sorry for not updating this thread sooner.  My problem was caused by the last iOS update.  It knocked my iPhone off iCloud.  I had to do Apple's whole ridiculous log out/in of iCloud routine from every device.  I could not log off iCloud from Mac for some unknown reason.  Deleting the keychain files manually finally released iCloud.

Handoff started working immediately after reboot.  Continuity did not start working until the next day... all by itself.  I really hope Apple makes these features more reliable and robust in future updates
